repository/c.json

```diff
@@ -2687,6 +2687,17 @@
 					"tags": true
 				}
 			]
+		},{
+			"name": "CompletePHP",
+			"details": "https://github.com/desean1625/CompletePHP",
+			"author": "Desean1625",
+			"labels": ["completions","PHP","Documentation","autocomplete"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```

Comment:
You can no longer use branch-based releases. You must instead use `"tags": true` and then create semver tags on your git repo to create a release.